crates/tauri-runtime-wry/src/lib.rs

```diff
@@ -3923,39 +3962,65 @@ fn create_window<T: UserEvent, F: Fn(RawWindow) + Send + 'static>(
     } else {
       event_loop.primary_monitor()
     };
-
     if let Some(monitor) = monitor {
+      let scale_factor = monitor.scale_factor();
       let desired_size = window_builder
         .inner
         .window
         .inner_size
         .unwrap_or_else(|| TaoPhysicalSize::new(800, 600).into());
-      let scale_factor = monitor.scale_factor();
-      #[allow(unused_mut)]
-      let mut window_size = window_builder
+      let mut inner_size = window_builder
         .inner
         .window
         .inner_size_constraints
         .clamp(desired_size, scale_factor)
         .to_physical::<u32>(scale_factor);
+      let mut window_size = inner_size;
+      #[allow(unused_mut)]
+      // Left and right window shadow counts as part of the window on Windows
+      // We need to include it when calculating positions, but not size
+      let mut shadow_width = 0;
```

Comment:
why do we need to include it? I think we should just include the starting of the window border not shadow

Reply:
If I remembered correctly, the shadow counts as part of the window when we set the positions, say your shadow is 5px left right bottom, and you set the window's position to (0, 0), then the actual window's top left will be placed at (5, 0